src/views/GameTemplates/GameTemplates.scss

```diff
@@ -0,0 +1,75 @@
+@import "../../index";
+
+.template-header {
+    display: flex;
+    color: $color-white;
+    font-size: 2rem;
+    justify-content: space-evenly;
+
+    .input-field {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-evenly;
+        padding: .5em;
+
+        input {
+            height: 50px;
+            width: 300px;
+            font-size: 1rem;
+            border: round;
+            border-color: white;
+            // border-radius: 10px;
+            box-shadow: 6px 6px 14px -5px rgba(0, 0, 0, 0.5);
+            background-color: transparent;
+            color: white;
```

Comment:
We can set the left and right padding within the input to keep the text from bumping right up against the border:
```
color: white;
padding-left: 0.5em;
padding-right: 0.5em;
```